automatic/dropbox/dropbox.nuspec

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>dropbox</id>
+    <version>0.0</version>
+    <title>Dropbox</title>
+    <authors>Dropbox</authors>
+    <owners>chocolatey,ferventcoder,the-running-dev</owners>
+    <licenseUrl>http://www.dropbox.com/terms</licenseUrl>
+    <projectUrl>https://www.dropbox.com/</projectUrl>
+    <iconUrl>https://raw.githubusercontent.com/chocolatey/chocolatey-coreteampackages/master/icons/dropbox.svg</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+    Dropbox is a free service that lets you bring all your photos, docs,
+    and videos anywhere. This means that any file you save to your
+    Dropbox will automatically save to all your computers,
+    phones and even the Dropbox website. Dropbox also makes it super easy
+    to share with others, whether you're a student or professional,
+    parent or grandparent. Even if you accidentally spill a latte on your
+    laptop, have no fear! You can relax knowing that Dropbox always has
+    you covered, and none of your stuff will ever be lost.
+    </description>
+    <summary>Dropbox is a free service that lets you bring all your photos, docs, and videos anywhere.</summary>
+    <tags>dropbox virtual files file admin freeware cross-platform</tags>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages</packageSourceUrl>
```

Comment:
the package source should be in the format `https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/{packageName}`